### ISSUE TITLE
ngAria disables tabbing to a radio set when no default value selected

### ISSUE THREAD

**Issue by OverZealous** [angular/angular.js#12492]
- **Overview of the Issue** - When a radio set is configured with no default selected value (ie: the value starts out as undefined), all the radio buttons will have their `tabindex` set to `-1`, making it impossible to tab to the radio set.
- **Motivation for or Use Case** - This makes it difficult to use a normal radio set for focus when you don't want to preselect the value for the user.
- **Angular Version** - 1.4.1
- **Browsers and Operating System** - Tested on FF39 on Mac OS X 10.10.4
- **Reproduce the Error** - [View on Plunkr](http://embed.plnkr.co/oYvOGsHxcYPMZOuRLQg1/preview)
- **Related Issues** - A partial attempt to fix this was issue #10212
- **Suggest a Fix** - I'm not sure what the right solution here is.  Some ideas:
    - I don't know what the purpose of setting the `tabindex` to `-1` is, if it's not critical, it probably should be removed.
    - Another option would be to find the entire set of radios with a matching `name`, and if none are selected, then all should be active, otherwise only the selected on is active.

**Comment by OverZealous** [comment]
Note: I added `tabindex="0"` to all of the radios, and tabbing worked as expected (it only tabbed to the selected radio once one was checked), so I feel there is no good reason to set `tabindex` to anything other than `0` if the input is enabled.

**Comment by marcysutton** [comment]
ngAria shouldn't be adding `tabindex` to radio inputs at all–I will be sure to account for that in #12262.

**Comment by marcysutton** [comment]
Closing as duplicate of #11500.

**Comment by mmaday** [comment]
@marcysutton This problem is still happening in Angular 1.4.7.  See the plunkr at http://plnkr.co/edit/f58uFdkxKNsHFcbIN1nm?p=preview.  The problematic code is in the aria ngModel directive:
```
          function getRadioReaction() {
            if (needsTabIndex) {
              needsTabIndex = false;
              return function ngAriaRadioReaction(newVal) {
                var boolVal = (attr.value == ngModel.$viewValue);
                elem.attr('aria-checked', boolVal);
                elem.attr('tabindex', 0 - !boolVal);
              };
```

**Comment by gkalpak** [comment]
Seems like a valid issue. WDYT, @marcysutton ?

**Comment by marcysutton** [comment]
Totally a bug! ngAria shouldn't be adding tabindex to native inputs at all. That should have gone into #11500.

**Comment by marcysutton** [comment]
@gkalpak I wrote a test that reproduced the problem and got a fix submitted in #13095.
